blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/AstyanaxReader.java

```diff
@@ -300,14 +302,18 @@ public String getLastStringValue(Locator locator) {
 
     public static String getUnitString(Locator locator) {
         String unitString = null;
-        try {
-            unitString = metaCache.get(locator, MetricMetadata.UNIT.name().toLowerCase(), String.class);
-        } catch (CacheException ex) {
-            log.warn("Cache exception reading unitString from MetadataCache: ", ex);
-        }
-        if (unitString == null) {
-            unitString = UNKNOWN;
+        // Only grab units from cassandra, if we have to
+        if (!Configuration.getInstance().getBooleanProperty(CoreConfig.USE_ES_FOR_UNITS)) {
+            try {
+                unitString = metaCache.get(locator, MetricMetadata.UNIT.name().toLowerCase(), String.class);
+            } catch (CacheException ex) {
+                log.warn("Cache exception reading unitString from MetadataCache: ", ex);
+            }
+            if (unitString == null) {
+                unitString = UNKNOWN;
```

Comment:
can we store a constant in the config file that is used when units are unknown ie CoreConfig.Unknown_Unit_String?
Just to make it consistent across the board.